src/test/java/org/carlspring/strongbox/janusgraph/reposiotries/ArtifactEntryRepositoryTest.java

```diff
@@ -92,10 +92,9 @@ public void crudShouldWork()
         assertEquals(createdOn, artifactEntrySaved.getCreated());
         Set<String> tags = artifactEntrySaved.getTags();
         assertNotNull(tags);
-        //TODO: fix tags
-        //assertEquals(2, tags.size());
-        //assertTrue(tags.contains("release"));
-        //assertTrue(tags.contains("stabile"));
+        assertEquals(2, tags.size());
+        assertTrue(tags.contains("release"));
+        assertTrue(tags.contains("stabile"));
```

Comment:
Cosmetic: There's a typo. It should be `stable`.

Reply:
Fixed